### ISSUE TITLE
[a11y][TABS]: keyboard navigation issues in Safari and Firefox

### ISSUE THREAD

**Issue by ginabak** [Shopify/polaris-react#3831]
## Issue summary

During the 🎩 of https://github.com/Shopify/web/pull/36467 **Bump polaris v5.13.0**, I noticed the following issues:

1. Keyboard navigation in `Tabs` seems to be problematic in Safari & FireFox (see GIFS and expand to see behaviour)
2. Safari: Cannot easily tab into any of the tabs via the keyboard
3. Firefox: Cannot tab into disclosure portion via the keyboard

## Expected behavior
One should be able to navigate tabs easily via the Keyboard. See w3 example [here](https://www.w3.org/TR/wai-aria-practices/examples/disclosure/disclosure-navigation.html). 
<details><summary> Expand for GIF </summary> 

![Chrome-Proper-Tabs](https://user-images.githubusercontent.com/43223543/104487401-0bba9080-559b-11eb-9895-b8cebb8dec4f.gif)

</details>

## Actual behavior

<details><summary> Expand for GIFs </summary>

### SAFARI BEHAVIOUR

![Safari-Tab-Issue](https://user-images.githubusercontent.com/43223543/104486657-0d378900-559a-11eb-9ce7-fa80a5a99b3d.gif)

### FIREFOX BEHAVIOUR

![Firefox-Tab-issues](https://user-images.githubusercontent.com/43223543/104486669-13c60080-559a-11eb-930f-112c8dbfdd9a.gif)

</details>

## Steps to reproduce the problem

(set up local environment) 
#### Safari behaviour
1. Open Safari
2. Go to https://shop1.myshopify.io/admin/products 
3. Ensure you have a disclosure, like this: 
<img width="715" alt="Screen Shot of Tabs with Disclosure Option" src="https://user-images.githubusercontent.com/43223543/104486996-8a62fe00-559a-11eb-884b-1328050da231.png">
4. Try to tab via the keyboard

#### FireFox behaviour
1. Open FireFox
2. Go to https://shop1.myshopify.io/admin/products 
3. Ensure you have a disclosure, like this: 
<img width="715" alt="Screen Shot of Tabs with Disclosure Option" src="https://user-images.githubusercontent.com/43223543/104486996-8a62fe00-559a-11eb-884b-1328050da231.png">
4. Try to tab via the keyboard

**Comment by ginabak** [comment]
🤦🏽 🤦🏽‍♂️ 🤦🏽‍♀️ 
ALL THE FACE PALMS because 💡 TIL 💡  when you are testing keyboard navigation (other than Chrome), you need to enable the following on MacOS: 

1. Go to: System Preferences ➡️ Keyboard ➡️ Shortcuts

Make sure `Use keyboard navigation to move focus between controls` is ENABLED! ✅ 
See screenshot below: 
<img width="669" alt="Screen Shot 2021-01-21 at 4 16 36 PM" src="https://user-images.githubusercontent.com/43223543/105416609-6772c280-5c08-11eb-975f-ddcd96138b1c.png">

2. Open Safari ➡️ Preferences ➡️ Advanced ➡️ Accessibility

Make sure `Press Tab to highlight each item on a webpage` is ENABLED! ✅ 
<img width="824" alt="Screen Shot 2021-01-21 at 4 46 11 PM" src="https://user-images.githubusercontent.com/43223543/105416613-6a6db300-5c08-11eb-930c-0c34b13c8bd5.png">

One concern I have is, is it appropriate for us to assume that all MacOS users that require keyboard navigation know to enable this? Do other developers testing accessibility know that (by default - in either a system upgrade, etc) that keyboard navigation is not set up?

Closing the issue as it is no longer an "issue" the above are set up properly 😬 😓
